balanced-optiml/bal-optiml-script/metadata.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Balanced OptiML Script",
+  "description": "Applying undersampling techniques for datasets that contain a rare class before building OptiMLs. The result is a list of OptiMLs built on a collection of undersampled datasets and the unbalanced test data to do the final evaluation.",
+  "kind": "script",
+  "imports": ["../bal-optiml-library"],
+  "source_code": "script.whizzml",
+  "inputs": [
+    {
+      "name": "ds-id",
```

Comment:
nit: for consistency with the other parameter names, none of which are abbreviated, perhaps this could be called "dataset" or "dataset-id".

Reply:
ok